viewer/packages/rendering/src/pipeline-executors/BasicPipelineExecutor.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { PipelineExecutor } from '../PipelineExecutor';
+import { RenderPipelineProvider } from '../RenderPipelineProvider';
+
+export class BasicPipelineExecutor implements PipelineExecutor {
```

Comment:
I think a unit test would be good, even if it simple (to encourage any future changes to add tests...)